### PR TITLE
Add page for myUplink

### PR DESCRIPTION
**Pull request by pajzo**
## Proposed change

Documentation for a new integration: myUplink

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/86522
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/myuplink.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ha_domain: myuplink
+ha_codeowners:
+  - '@pajzo'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The myUplink integration allows users to get information about their devices supporting myUplink using the [official cloud API](https://dev.myuplink.com).
+
+Depending on the type of devices in your system there will be added one or more sensor entities to Home Assistant. Currently there is support for generic and temperature sensors.
```
Comment (frenck) [inline source/_integrations/myuplink.markdown]:
```suggestion
Depending on the type of devices in your system, one or more sensor entities to Home Assistant. Currently, there is support for generic and temperature sensors.
```

File: source/_integrations/myuplink.markdown [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ha_iot_class: Cloud Polling
+ha_release: '2023.2'
+ha_domain: myuplink
+ha_codeowners:
+  - '@pajzo'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The myUplink integration allows users to get information about their devices supporting myUplink using the [official cloud API](https://dev.myuplink.com).
```
Comment (frenck) [inline source/_integrations/myuplink.markdown]: We write pages for the reader, not a group of people.

```suggestion
The myUplink integration lets you get information about the devices supporting myUplink using the [official cloud API](https://dev.myuplink.com).
```

File: source/_integrations/myuplink.markdown [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Depending on the type of devices in your system there will be added one or more sensor entities to Home Assistant. Currently there is support for generic and temperature sensors.
+
+## Prerequisites
+
+1. Visit [https://myuplink.com/register](https://myuplink.com/register) and sign up for a user account.
+2. Go to [Applications](https://dev.myuplink.com/apps), register a new App:
```
Comment (frenck) [inline source/_integrations/myuplink.markdown]:
```suggestion
2. Go to [**Applications**](https://dev.myuplink.com/apps), register a new App:
```

File: source/_integrations/myuplink.markdown [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+- Application ID: Home Assistant (or whatever name makes sense to you)
+- OAuth Flow: Authorization Code Grant Flow
+- Redirect URI: `https://my.home-assistant.io/redirect/oauth`
```
Comment (frenck) [inline source/_integrations/myuplink.markdown]:
```suggestion
- **Application ID**: Home Assistant (or whatever name makes sense to you)
- **OAuth Flow**: Authorization Code Grant Flow
- **Redirect URI**: `https://my.home-assistant.io/redirect/oauth`
```

File: source/_integrations/myuplink.markdown [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+The `<HOME_ASSISTANT_URL>` must be the same as used during the configuration/
+authentication process.
+
+Internal examples: `http://192.168.0.2:8123/auth/external/callback`, `http://homeassistant.local:8123/auth/external/callback`." 
+
+{% enddetails %}
+
+{% include integrations/config_flow.md %}
+
+The integration configuration will ask for the *Client ID* and *Client Secret* created above. See [Application Credentials](/integrations/application_credentials) for more details.
```
Comment (frenck) [inline source/_integrations/myuplink.markdown]:
```suggestion
The integration configuration will require the **Client ID** and **Client Secret** created above. See [Application Credentials](/integrations/application_credentials) for more details.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Some tiny tweaks, otherwise it looks great 👍 

../Frenck

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/myuplink.markdown [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+title: myUplink
+description: Instructions on how to set up the myUplink integration within Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: '2023.2'
```
Comment (MartinHjelmare) [inline source/_integrations/myuplink.markdown]:
```suggestion
ha_release: '2024.2'
```

File: source/_integrations/myuplink.markdown [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+ha_domain: myuplink
+ha_codeowners:
+  - '@pajzo'
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The myUplink integration lets you get information about the devices supporting myUplink using the [official cloud API](https://dev.myuplink.com).
+
+Depending on the type of devices in your system, one or more sensor entities to Home Assistant. Currently, there is support for generic and temperature sensors.
```
Comment (MartinHjelmare) [inline source/_integrations/myuplink.markdown]:
```suggestion
Depending on the type of devices in your system, one or more sensor entities will be created in Home Assistant. Currently, there is support for generic and temperature sensors.
```

Comment (MartinHjelmare) [reply source/_integrations/myuplink.markdown]: Generic sensors sounds weird. Maybe just don't mention what sensors are supported as that will change over time and it's hard to keep the documentation updated to cover it exactly?

File: source/_integrations/myuplink.markdown [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+## Prerequisites
+
+1. Visit [https://myuplink.com/register](https://myuplink.com/register) and sign up for a user account.
+2. Go to [**Applications**](https://dev.myuplink.com/apps), register a new App:
```
Comment (MartinHjelmare) [inline source/_integrations/myuplink.markdown]:
```suggestion
2. Go to [**Applications**](https://dev.myuplink.com/apps), and register a new App:
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**